Strategies/IITB_KPIs.md

```diff
@@ -30,13 +30,26 @@ Toil is the kind of work tied to running a production service that tends to be m
 
 The indicator will be measured two ways:
 
-1. For toil in IITB Services, using ??? and measure the ratio of X over Y.
+1\. Toil in IITB Services
 
-`ratio = X / Y`
+We can measure both the service and development sides:
 
-2. For toil in ESDC Services, using the [ESDC's Service Catalogue](https://www.canada.ca/en/employment-social-development/corporate/portfolio/service-canada/programs.html) and measure the ratio of service requests to amount of personnel.
+For service requests (e.g. Helpdesk):
 
-`ratio = service requests / service personnel`
+`service ratio = service requests / service personnel`
+
+For the development side, there would be multiple metrics, some of them being influenced by the evolution toward DevOps, e.g.:
```

Comment:
I would suggest we use the standard Devops mertrics (we can review the annual DevOps reports by Dora and Puppet [the most popular]), but I also took notes [here](onenote:http://dialogue/grp/BU6810070/Shared%20Documents/IT%20Strategy%20Workbook/Notes.one#DORA%20-%20State%20of%20DevOps,%202018&section-id={8F7042FC-8CA2-4132-A942-B030364A6FE4}&page-id={A9CF60E1-957B-4421-A0CC-E2360B5B16D2}&end). About half way down you will see a grid (46 times more frequent code deployments, etc...) 

For development, sec & ops, I would suggest using these metrics